### Retrieve chess_server game play history. 
#
#

---
ChessColor[]  player  # player [White, Black]
string[]      SAN     # move in Standard Algebraic Notation
